docs/templates/template_reference.md

```diff
@@ -419,6 +419,7 @@ The only way to remediate is to recompile and reinstall the kernel, so no remedi
     - **value** - The value the configuration should have
       When **value** is `"n"`, the check will pass when the config is absent, commented out or has the
       value `n` in the `/boot/config-*` files.
+    - **variable** - The variable to get the value from.
```

Comment:
```suggestion
    - **variable** - The variable to get the value from. Cannot be used if value is set and vice-versa.
```
Please add something like this to explain that both parameter can't be used at the same time.